shopfloor/tests/test_cluster_picking_base.py

```diff
@@ -35,8 +35,8 @@ def _line_data(self, move_line, qty=None, package_dest=False, qty_done=None, **k
             data["package_dest"] = None
         if qty:
             data["quantity"] = qty
-        if qty_done:
-            data["qty_done"] = qty_done
+        if qty_picked:
+            data["qty_picked"] = qty_picked
```

Comment:
We should probably not change `qty_done` here, it's data we are sending to the client.